modules/openapi-generator/src/main/resources/r/modelGeneric.mustache

```diff
@@ -238,7 +238,11 @@
           self$`{{name}}`
           {{/isPrimitiveType}}
           {{^isPrimitiveType}}
-          self$`{{name}}`$toJSON()
+          if (length(names(self$`{{name}}`$toJSON())) == 0L && is.character(jsonlite::fromJSON(self$`{{name}}`$toJSON()))) {
```

Comment:
i saw you using is.character check

Does it also cover numeric values such as 13, 5.6  ?